### PR TITLE
MGMT-19003: Add CI job to test 4, 5 control plane node cluster installation

### PR DESCRIPTION
**Pull request by danmanor**
Adding jobs to test OCP installation with 4/5 control planes

### CODE REVIEW

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml [hunk 1]
```diff
@@ -1390,6 +1390,50 @@ tests:
       ASSISTED_CONFIG: |
         PLATFORM=external
     workflow: assisted-baremetal
+- as: e2e-metal-assisted-5-control-planes-4-17
```
Comment (adriengentil) [inline ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml:1393]:
```suggestion
- always_run: false
  as: e2e-metal-assisted-5-control-planes-4-17
```

I don't think we want to run these jobs each time a PR is opened?

Comment (danmanor) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml]: Yes thank you !

**Review by adriengentil — APPROVED** [review]

**Pull request merged**